dev/test/device_file_io.cu

```diff
@@ -0,0 +1,61 @@
+/*
+Tests device <-> file IO functions
+
+compile and run as (from dev/test directory)
+nvcc -o device_file_io device_file_io.cu && ./device_file_io
+*/
+
+
+#include "../../llmc/cuda_common.h"
+#include <vector>
+#include <random>
+#include <algorithm>
+
+void test(size_t nelem, size_t wt_buf, size_t rd_buf) {
```

Comment:
nit: got confused with what these are until i analyzed `device_to_file` function and realized these are sizes
maybe rename to `wt_buf_size` and `rd_buf_size`?

Reply:
yes, good idea